packages/create-qwik/create-interactive.ts

```diff
@@ -114,24 +124,25 @@ export async function runCreateInteractiveCli() {
 
   if (gitInitAnswer) {
     if (fs.existsSync(join(outDir, '.git'))) {
-      console.log(`Git has already been initialized`);
-      console.log(``);
-      return;
-    }
-
-    s.start('Git initializing...');
-
-    try {
-      await runCommand('git', ['init'], outDir).install;
-      await runCommand('git', ['add', '-A'], outDir).install;
-      await runCommand('git', ['commit', '-m', 'Initial commit ⚡️'], outDir).install;
+      log.info(`Git has already been initialized before. Skipping...`);
+    } else {
+      s.start('Git initializing...');
+
+      try {
+        const res = [];
+        res.push(await runCommand('git', ['init'], outDir).install);
+        res.push(await runCommand('git', ['add', '-A'], outDir).install);
+        res.push(await runCommand('git', ['commit', '-m', 'Initial commit ⚡️'], outDir).install);
+
+        if (res.some((r) => r === false)) {
+          throw '';
```

Comment:
the promises above do not throw but `resolve(false)` which is not catchable. this is a workaround. if you @adamdbradley think we could safely refactor the `runCommand` fn to throw, we could get rid of these workaround 👼